portfolio/src/main/java/com/google/sps/servlets/AuthServlet.java

```diff
@@ -0,0 +1,31 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+
+@WebServlet("/auth")
+public class AuthServlet extends HttpServlet{
+    
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse res) throws IOException{
+        if(!ServletUtil.USERSERVICE.isUserLoggedIn()){
+            String urlToRedirectToAfterUserLogsIn = ServletUtil.COMMENTS_HTML;
```

Comment:
no need to create this local variable. Just replace it's usage with ServletUtil.COMMENTS_HTML